### PR TITLE
US106395 set input type to search

### PR DESCRIPTION
**Pull request by ciss1995**

### CODE REVIEW

File: d2l-input-search.js [hunk 1]
```diff
@@ -33,29 +33,29 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-input-search">
 				width: 100%
 			}
 
-			input[type="text"].d2l-input {
+			input[type="seach"].d2l-input {
```
Comment (svanherk) [inline d2l-input-search.js]: Typo!  Should be "search" 😛

Comment (dlockhart) [reply d2l-input-search.js]: Oh wow good catch!

Comment (dlockhart) [reply d2l-input-search.js]: Really missing those perceptual diff tests...

Comment (dlockhart) [reply d2l-input-search.js]: @ciss1995 I'll manually fix this and re-release.

**Pull request merged**